doc/releases/changelog-dev.md

```diff
@@ -33,42 +33,116 @@
+  template.
   [(#5107)](https://github.com/PennyLaneAI/pennylane/pull/5107)
 
-* Create the `qml.Reflection` operator, useful for amplitude amplification and its variants.
-  [(#5159)](https://github.com/PennyLaneAI/pennylane/pull/5159)
+  The usage of `qml.FABLE` is similar to `qml.BlockEncode` but provides a more
+  efficient circuit construction at the cost of a user-defined approximation 
+  level, `tol`. The number of wires that `qml.FABLE` operates on is `2*n + 1`, 
+  where `n` defines the dimension of the $2^n \times 2^n$ matrix that we want to 
+  block-encode.
 
   ```python
-  @qml.prod
-  def generator(wires):
-        qml.Hadamard(wires=wires)
-
-  U = generator(wires=0)
-
+  A = np.random.random((2, 2)) # n = 1, 2*n + 1 = 3
   dev = qml.device('default.qubit')
+
   @qml.qnode(dev)
   def circuit():
+      qml.FABLE(A, tol = 0.1, wires=range(3))
+      return qml.state()
+  ``` 
+
+  ```pycon
+  >>> circuit()
+  tensor([0.29906437+0.j, 0.20579772+0.j, 0.29906437+0.j, 0.20579772+0.j,
+          0.40070002+0.j, 0.45568333+0.j, 0.40070002+0.j, 0.45568333+0.j], requires_grad=True)
+  ```
 
-        # Initialize to the state |1>
-        qml.PauliX(wires=0)
+* Reflecting about a given quantum state is now available via `qml.Reflection`.
+  This operation is very useful in, say, the amplitude amplification algorithm.
+  [(#5159)](https://github.com/PennyLaneAI/pennylane/pull/5159)
 
-        # Apply the reflection
-        qml.Reflection(U)
+  `qml.Reflection` works by providing an operation, $U$, that *prepares* the 
+  desired state, $\vert \psi \rangle$, that we want to reflect about. In other 
+  words, $U$ is such that $U \vert 0 \rangle = \vert \psi \rangle$. In 
+  PennyLane, $U$ must be an `Operator`.
+  
+  For example, if we want to reflect about 
+  $\vert \psi \rangle = \vert + \rangle$, then $U = H$:
 
-        return qml.state()
+  ```python
+  U = qml.Hadamard(wires=wires)
 
+  dev = qml.device('default.qubit')
+  @qml.qnode(dev)
+  def circuit():
+        qml.Reflection(U)
+        return qml.state()
   ```
 
   ```pycon
   >>> circuit()
-  tensor([1.+6.123234e-17j, 0.-6.123234e-17j], requires_grad=True)
+  tensor([0.-6.123234e-17j, 1.+6.123234e-17j], requires_grad=True)
   ```
+
+  For cases where $U$ comprises many operations, you can formulate it as an 
+  `Operator` by 
   
-* The `qml.AmplitudeAmplification` operator is introduced, which is a high-level interface for amplitude amplification and its variants.
+  * creating a user-defined 
+    [custom operation](https://docs.pennylane.ai/en/stable/development/adding_operators.html) 
+    with a decomposition, or by
+
+  * creating a quantum function containing each operation, one per line, then 
+    decorate the quantum function with `@qml.prod`, which turns the quantum 
+    function into a `Prod` instance:
+
+    ```python
+    @qml.prod
+    def U(wires):
+        qml.Hadamard(wires=wires[0])
+        qml.RY(0.1, wires=wires[1])
```

Comment:
Minor point but it felt like the `wires` was a bit of a distraction, maybe we just make it have fixed wires?

Reply:
Good point!